collections/Lyell/MS_Lyell_50.xml

```diff
@@ -78,39 +85,147 @@
                      </altIdentifier>
                   </msIdentifier>
                   <msContents>
-                     <msItem xml:id="MS_Lyell_50-part1-item1" n="1" class="#theologica">
-                        <title type="desc" key="work_11089">Collection of quaestiones and sentences from the school of <persName role="aut" key="person_77109407">
-                              Anselm of
-                           Laon
-                           </persName>.</title>
-                        <textLang mainLang="la">Latin</textLang>
+                     <textLang mainLang="la">Latin</textLang>
+                     <msItem>
```

Comment:
I think it would be worth using `@n` to preserve the numbering given by de la Mare - e.g. if someone wanted to cite her description by saying 'art. 1g'.